qiskit/visualization/circuit/latex.py

```diff
@@ -579,7 +579,26 @@ def _build_symmetric_gate(self, op, gate_text, wire_list, col):
     def _build_measure(self, node, col):
         """Build a meter and the lines to the creg"""
         wire1 = self._wire_map[node.qargs[0]]
-        self._latex[wire1][col] = "\\meter"
+        if node.op.basis:
+            # Modification of `\meter` in https://www.ctan.org/tex-archive/graphics/qcircuit
+            # pylint: disable=invalid-name
+            PAULI_METER = """
+                *=<1.8em,1.4em>
+                {
+                    \\xy ="j",
+                    "j"-<.778em,.322em>;{"j"+<.778em,-.322em> \\ellipse ur,_{}},
+                    "j"-<0em,.4em>;p+<.5em,.9em> **\\dir{-},
+                    "j"+<2.2em,2.2em>*{},"j"-<2.2em,2.2em>*{},
+                    <-.5em,.4em> *\\txt{\\tiny{%s}}
+                    \\endxy
+                } 
+                \\POS ="i",
+                "i"+UR;"i"+UL **\\dir{-};"i"+DL **\\dir{-};"i"+DR **\\dir{-};"i"+UR **\\dir{-},
```

Comment:
(On my phone so I can't select properly.)

This looks like `qcircuit` code. That's GPL licensed, so we legally cannot include it, or code derived from it, in Qiskit. We need to be really careful when copying code blocks like this - attribution is not generally sufficient. It's legally tricky for us now even if you try and write your own implementation, because it's hard to be certain that it wouldn't be a derived work.

Perhaps we can use their `\measure` macro somehow?

Reply:
Ah! Good catch, I have definitely derived it from their code. 

I did try to use their macro but it proved to be very challenging so I dropped down to base `xypic` code and recreated part of what they where doing adding the extra pieces. I have reached out to them to see if I can contribute this to their package.

Not sure how to address this otherwise. I'll think about it, thanks for bringing this up.

Reply:
I [opened a PR](https://github.com/CQuIC/qcircuit/pull/47) to contribute this piece of code to `qcircuit`